t/cobrand/northamptonshire.t

```diff
@@ -246,6 +246,41 @@ subtest 'check further investigation state' => sub {
 
     $mech->content_contains('Investigating');
     $mech->content_lacks('Under further investigation');
+
+    $comment->set_extra_metadata('external_status_code' => '');
+    $comment->update;
+    my $comment2 = FixMyStreet::DB->resultset('Comment')->create( {
+        mark_fixed => 0,
+        user => $user,
+        problem => $report,
+        anonymous => 0,
+        text => 'this is a comment',
+        confirmed => DateTime->now,
+        state => 'confirmed',
+        problem_state => 'investigating',
+        cobrand => 'default',
+    } );
+    $comment2->set_extra_metadata('external_status_code' => 'further');
+    $comment2->update;
+
+    FixMyStreet::override_config {
+        ALLOWED_COBRANDS => [ { northamptonshire => '.' } ],
+        MAPIT_URL => 'http://mapit.uk/',
+        COBRAND_FEATURES => {
+            extra_state_mapping => {
```

Comment:
I don't understand what this is for.

Reply:
It's to enable us to display a status on the front end based on a state and external status code. This is to handle things like the further investigation state that NCC want, to which they've added a few more.

Reply:
Okay, but the code to do that isn't in this PR or on master? :)

Reply:
Ignore me!